global/prometheus-alertmanager-operated/templates/am-config-route-slack.yaml

```diff
@@ -476,9 +476,10 @@ spec:
           - name: severity
             matchType: "="
             value: critical
-          - name: region
-            matchType: "=~"
-            value: {{ without .Values.regions "qa-de-1" | join "|" }}
+          # Removed for testing purpose
+          # - name: region
+          #   matchType: "=~"
+          #   value: {{ without .Values.regions "qa-de-1" | join "|" }}
```

Comment:
This would remove the region variable entirely. We want all to be joined with `|`.